sample_files/nginx.conf

```diff
@@ -47,7 +47,7 @@ server {
     # This location name should be set to DMOJ_PDF_PROBLEM_INTERNAL.
     #location /pdfcache {
     #    internal;
-    #    root <path to pdf cache directory, without the final /pdfcache>;
+    #    root <the value of DMOJ_PDF_PROBLEM_CACHE in local_settings.py>;
```

Comment:
Wrong since this is `root` and not `alias`, so the full URL is appended to the URL.